helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -67,6 +67,7 @@ rules:
   - ""
   resources:
   - configmaps
+  - secrets
```

Comment:
The permissions here are the same as for confgmaps but they are wide. But I don't think we can restrict by namespace because we need to manage secrets in multiple namespaces. WDYT?

cc @puja108